src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2327,5 +2346,104 @@ public static void SetSize(IntPtr window, int portraitWidth, int portraitHeight,
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Sets the processKeyWithKeycode action.
+        /// If the key event is from the external device, DeviceInfo will have its name, class, and subclass information.
+        /// </summary>
+        /// <param name="processKeyWithKeycode">
+        /// The action is called when the key event is received from the external devices or the SendKey function.
+        /// This Event processes the key event before an associated text input UI control does.
+        /// </param>
```

Comment:
Please include some information about meaning of the callback return boolean. Does it affect anything?

Reply:
I added description for return boolean.